test/jdk/java/net/NetworkInterface/UniqueMacAddressesTest.java

```diff
@@ -21,87 +21,66 @@
  * questions.
  */
 
-import java.net.InetAddress;
+import java.io.PrintStream;
+import java.io.UncheckedIOException;
 import java.net.NetworkInterface;
 import java.net.SocketException;
 import java.util.ArrayList;
 import java.util.Arrays;
-import java.util.Enumeration;
 import java.util.List;
+import java.util.stream.Collectors;
 
+import jdk.test.lib.NetworkConfiguration;
 
 /*
  * @test
  * @bug 8021372
  * @summary Tests that the MAC addresses returned by NetworkInterface.getNetworkInterfaces are unique for each adapter.
- *
+ * @library /test/lib
+ * @build jdk.test.lib.NetworkConfiguration
+ * @run main/othervm UniqueMacAddressesTest
  */
 public class UniqueMacAddressesTest {
 
+    static PrintStream log = System.err;
+    record NetIfPair(String interfaceName, byte[] address) {}
```

Comment:
Nit: maybe add a blank line, and a comment:
```
    static PrintStream log = System.err;

    // A record pair of (NetworkInterface::name,  NetworkInterface::hardwareAddress)
    record NetIfPair(String interfaceName, byte[] address) {}
```

Reply:
I agree, this is more clear for some one new looking at the test. Will add in that comment now